File: 아기호랑이반/1주차 문제/박혁진/거스름돈.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int price = Integer.parseInt(sc.nextLine()); // 지불가

        int change = 1000 - price; // 거스름돈
        int coinCount = 0; // 동전 개수
        int[] coins = {500, 100, 50, 10, 5, 1}; // 동전 단위 배열

        for (int coin : coins) { 
            if (change >= coin) {
                coinCount += change / coin;
                change %= coin;
            }
        }



        System.out.println(coinCount);
    }
}

File: 아기호랑이반/1주차 문제/김효정/거스름돈.c
#include <stdio.h>

int main(void)
{

    int price, change;
    int coins[] = {500, 100, 50, 10, 5, 1};

    // 물건값(price) 입력
    scanf("%d", &price);

    // 거스름돈 총액 = 1000엔을 내고 물건값(price)을 제외한 금액
    change = 1000 - price;

    int cnt = 0;

    for (int i = 0; i < 6; i++)
    {
        int coin = coins[i];
        cnt += change / coin;
        change %= coin;
    }

    printf("%d\n", cnt);
    return 0;
}
